Dockerfile.v4.2.x

```diff
@@ -1,45 +1,21 @@
-FROM ubuntu:18.04
+ARG BASE=ubuntu:18.04
+FROM ${BASE}
+
 LABEL maintainer="Matt Bentley <mbentley@mbentley.net>"
 
-ARG DEBIAN_FRONTEND=noninteractive
 ARG OMADA_VER=4.2.8
 ARG OMADA_TAR="Omada_SDN_Controller_v${OMADA_VER}_linux_x64.tar.gz"
 ARG OMADA_URL="https://static.tp-link.com/2020/202012/20201211/${OMADA_TAR}"
+# valid values: amd64 (default) | arm64 | armv7l
+ARG ARCH=amd64
+# suffix to {entrypoint,install}.sh
+ARG SCRIPT_VER=4.2.x
```

Comment:
It may not be worth making this an argument as I was actually thinking about combining the two entrypoints into a single shell script and have them do things different depending on the version.  I did the same thing with [the timemachine image's entrypoint script](https://github.com/mbentley/docker-timemachine/blob/master/entrypoint.sh) as there was enough in common to make it worth having just one file to maintain.  In that case, it is at the cost of adding complexity to a single file vs having to know what file to deal with.  It's complexity one way or another I suppose though.

Reply:
Yes, I added it only to stick the existing `entrypoint-4.2.x.sh`, but if you are planing to have a unique `entrypoint.sh` it makes sense to have only `install.sh` as well.

Reply:
I addressed some of your comments in https://github.com/aeschbacher/docker-omada-controller/commit/1a6c1560ce6d2ecbc9b142a94ca411f8f74d3a40 , and let you decide for `install.sh` vs `Dockerfile`

Reply:
Sounds good.